tasks/change-granule-collection-s3/schemas/output.json

```diff
@@ -0,0 +1,129 @@
+{
+  "title": "ChangeGranuleCollectionS3Output",
+  "description": "Describes the output produced by the move-granules task",
+  "type": "object",
+  "properties": {
+    "granules": {
+      "type": "array",
+      "description": "array of modified target granules",
+      "items": {
+        "type": "object",
+        "required": [
+          "granuleId",
+          "files"
+        ],
+        "properties": {
+          "granuleId": {
+            "type": "string"
+          },
+          "files": {
+            "type": "array",
+            "items": {
+              "additionalProperties": false,
+              "type": "object",
+              "required": [
+                "bucket",
+                "key"
+              ],
+              "properties": {
+                "bucket": {
+                  "description": "Bucket where file is archived in S3",
+                  "type": "string"
+                },
+                "checksum": {
+                  "description": "Checksum value for file",
+                  "type": "string"
+                },
+                "checksumType": {
+                  "description": "Type of checksum (e.g. md5, sha256, etc)",
+                  "type": "string"
+                },
+                "fileName": {
+                  "description": "Name of file (e.g. file.txt)",
+                  "type": "string"
+                },
+                "key": {
+                  "description": "S3 Key for archived file",
+                  "type": "string"
+                },
+                "size": {
+                  "description": "Size of file (in bytes)",
+                  "type": "number"
+                },
+                "source": {
+                  "description": "Source URI of the file from origin system (e.g. S3, FTP, HTTP)",
+                  "type": "string"
+                },
+                "type": {
+                  "description": "Type of file (e.g. data, metadata, browse)",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "oldGranules": {
```

Comment:
I think there's still an open question re: this output with respect to what we need downstream, but that can probably be modified as part of that work if needed.    Commenting mostly as a bookmark to have that conversation.

Reply:
Agreed to punt this to S3 cleanup task.